### ISSUE TITLE
Improve visuals of sub tabs in creator panel

### ISSUE THREAD

**Issue by manuhabitela** [gristlabs/grist-core#1299]
This is a task taken from the [right panel tabs issue](https://github.com/gristlabs/grist-core/issues/1011#issuecomment-2450181038) that should be worked on independently.

The idea is to improve the visuals of subtabs, as it feels better to have inactive tabs not being colored with the primary color: 

![Image](https://github.com/user-attachments/assets/a3d8d7b3-7cd3-4eb0-9f15-c4424e472d43)

@lusebille mockups: https://www.figma.com/design/wcpetFt6aOKzTszcvPPWLQ/%5B05%2F24%5D-Grist-Design?node-id=2761-141496&t=qX4iHFYd8uzsW8kZ-1

**Comment by manuhabitela** [comment]
Actually, working on the new colors first is required here, if I use the current grays they will not be visible, and If i use the new grays just in this issue it'll feel weird to have just those things having colors different from the main theme…

Waiting for feedback on https://github.com/gristlabs/grist-core/issues/1295 first.
